//IN THIS CODE AM IMPLEMENING THE PANIC
package main
import "fmt"
 func main (){
 fmt.Println("start the program")
 panic("A severe Error occured!")
 fmt.Println("Ending the program")
 }
